adafruit_wiznet5k/adafruit_wiznet5k_socket.py

```diff
@@ -385,13 +514,18 @@ def recv_into(self, buf, nbytes=0, flags=0):
         buf[:nbytes] = ret
         return nbytes
 
-    def recvfrom_into(self, buf, nbytes=0, flags=0):
-        """Reads some bytes from the connected remote address info the provided buffer.
+    def recvfrom_into(
+        self, buf: bytearray, nbytes: int = 0, flags: Any = 0
```

Comment:
another place I think we should match `flags` type declaration with the docstrings.